DOCS: lightspeed login text (#985)
MultiSafepay/docs: content/integrations/hosted/lightspeed_app/faq/how-to-login.md

---
title : "How to login"
meta_title: "Lightspeed plugin - Can I change the API key after installation? - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

> In early 2021 we have changed how our login system works for the MultiSafepay Payment app for lightspeed eCom. Our old system was password based, our new system uses the Single Sign On provided by lightspeed. This means you can directly login via Lightspeed eCom store to our settings page of the app.

Reviewer: In early 2021, we have changed how our login system works for the MultiSafepay Payment app for Lightspeed eCom. Our old system was password based, although now our new system uses the Single Sign On provided by Lightspeed. This means you can directly login via Lightspeed eCom store to our settings page of the app.

> To login using the new system you have to use our app store page. Login to the /admin section of your lightspeed store. Go to apps, my apps / "aangeschafte apps" and click on the name of our app, MultiSafepay Payments. This should open the app store page. Alternatively paste "/admin/store/apps/1517" after the base url your store.

Reviewer: To login using the new system you have to use our app store page. Login to the /admin section of your Lightspeed store. Go to apps, my apps / "aangeschafte apps" and click on the name of our app, MultiSafepay Payments. This should open the app store page. Alternatively paste "/admin/store/apps/1517" after the base url your store.

> On the store page click on "Goto app" / "Ga naar app" And you will be able to access your settings page. 

Reviewer: On the store page click on "Goto app" / "Ga naar app" and you will be able to access your settings page.

---
title : "How to login"
meta_title: "Lightspeed plugin - Can I change the API key after installation? - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

> In early 2021, we have changed how our login system works for the MultiSafepay Payment app for Lightspeed eCom. Our old system was password based, although now our new system uses the Single Sign On provided by Lightspeed. This means you can directly login via Lightspeed eCom store to our settings page of the app.

Reviewer 2: ```suggestion
In early 2021, we have changed the login method for the MultiSafepay Payment app for Lightspeed eCom. The old login method was password-based, while the new method uses the single sign-on provided by Lightspeed. This means you can directly log in via the Lightspeed eCom store to our settings page of the app.
```
login = noun
log in =  verb

Author: 👍 ty

> To login using the new system you have to use our app store page. Login to the /admin section of your Lightspeed store. 

Reviewer 2: ```suggestion
To log in using the new method you have to use our app store page. Login to the /admin section of your Lightspeed store. 
```
